pkg/sql/opt/exec/execbuilder/testdata/interleaved

```diff
@@ -58,7 +58,7 @@ EXPLAIN SELECT * FROM level4 WHERE k1 > 1 AND k1 < 3
 scan  ·              ·
 ·     missing stats  ·
 ·     table          level4@primary
-·     spans          /2/#/54/1/#/55/1-/2/#/54/1/#/55/2
+·     spans          [/2 - /2]
```

Comment:
I think you should use the verbose spans here, as these are trying to look at the interleaved keys